### PR TITLE
MAT-2555 update parsers to use mongoid models v0.0.4

### PR DESCRIPTION
**Pull request by adongare**
MAT-2555 update parsers to use mongoid models v0.0.4

Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/measure-loader/value_set_helpers_test.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ def test_code_systems
     assert code_systems != nil
     assert_equal 87, code_systems['by_name'].keys.size
     assert_equal 87, code_systems['by_name'].values.size
-    assert_equal 79, code_systems['by_oid'].keys.size
-    assert_equal 79, code_systems['by_oid'].values.size
+    assert_equal 81, code_systems['by_oid'].keys.size
+    assert_equal 81, code_systems['by_oid'].values.size
```
Comment (adongare) [inline test/unit/measure-loader/value_set_helpers_test.rb:19]: not sure how did this number changed. Is there any edits to spreadsheets

File: test/vcr_setup.rb [hunk 2]
```diff
@@ -14,7 +15,7 @@
   ENV['VSAC_API_KEY'] = "vcrpass" unless ENV['VSAC_API_KEY']
 
   # Ensure plain text passwords do not show up during logging
-  c.filter_sensitive_data('<VSAC_API_KEY>') {URI.escape(ENV['VSAC_API_KEY'])}
+  c.filter_sensitive_data('<VSAC_API_KEY>') { Addressable::URI.encode(ENV['VSAC_API_KEY']) }
```
Comment (adongare) [inline test/vcr_setup.rb:18]: removed deprecated method call for ruby 2.7

**Review by AndrewBird81 — APPROVED** [review]
